Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -433,44 +463,60 @@ def test_module(client: Client):
     demisto.results('ok')
 
 
-def ips_reputation_command(client: Client, ip, threshold=None, status="active,inactive"):
+def ips_reputation_command(client: Client, ip, threshold=None, status="active"):
```

Comment:
You moved the status input to the params, and yet you only send the args to these commands.